roles/st2/tasks/user.yml

```diff
@@ -51,3 +50,10 @@
     option: ssh_key_file
     value: "{{ _user.ssh_key_file }}"
     backup: yes
+
+- name: Disable requiretty
+  become: yes
+  replace:
+    dest: "/etc/sudoers"
```

Comment:
From the http://docs.ansible.com/ansible/replace_module.html can we use "recommended" `path`, instead of old aliases: `dest, destfile and name` which could become deprecated in future.